odfuzz/generators.py

```diff
@@ -3,120 +3,223 @@
 import random
 import uuid
 import datetime
+import base64
+import time
 
 from odfuzz.constants import BASE_CHARSET, HEX_BINARY
 from odfuzz.encoders import EncoderMixin
 
-START_DATE = datetime.datetime(1900, 1, 1, 0, 0, 0)
+START_DATE = datetime.datetime(1970, 1, 1, 0, 0, 0)
```

Comment:
We can go even farther: "Represents date and time with values ranging from 12:00:00 midnight, January 1, 1753 A.D. through 11:59:59 P.M, December 9999 A.D." (https://www.odata.org/documentation/odata-version-2-0/overview/)

Reply:
Irrelevant of @lubosmj comment, which in itself is valid - I do not understand at all the reason of the change from 1900 to 1970.

Reply:
Agreed, but OData JSON format considers 1/1/1970 midnight as the epoch [](https://www.odata.org/documentation/odata-version-2-0/json-format/) and having 2 different epochs for generating a synced pair of values is challenging. The ceiling is still 11:59:59 P.M, December 9999 A.D, so the overall change in range is minuscule.

Reply:
If I understand this correctly, this is in fact another problem raising from the fact that there is Odata:Core specification, Odata - Atom format and Odata - JSON format; and this PR is adding POST/PUT body support, implicitly expecting only JSON format output.

The Edm.DateTime differences are exactly the two different links you provided.

Reply:
as one hardcoded value, the 1970 is neccessary for the JSON output. But we uncovered something important here - so I created issue #80 just for this problem.

Reply:
> 
> 
> If I understand this correctly, this is in fact another problem raising from the fact that there is Odata:Core specification, Odata - Atom format (for Edm.DateTime same as core) and Odata - JSON format (different epoch); and this PR is adding POST/PUT body support, implicitly expecting only JSON format output.
> 
> The Edm.DateTime differences are exactly the two different links you provided.

yes, as of this PR, JSON output is implicitly expected. While generating datetime in different formats and epoch is not a problem, yet generating synced values across different formats and epoch needs some more effort, and could be a topic for future PR